export/export_pipeline/compute_gltf_representation.py

```diff
@@ -0,0 +1,270 @@
+from collections import namedtuple
+import json
+import os
+import time
+import unreal
+import yaml
+
+
+osp = os.path
+ss = unreal.get_editor_subsystem(unreal.UnrealEditorSubsystem)
+world = ss.get_editor_world()
+
+
+TRANS_UE_TO_MUJOCO_SCALE = 1.0  # keep cm
+ROT_UE_TO_MUJOCO_SCALE = 1.0  # keep degrees
+
+
+def export_meshes(actor, export_dir):
+  unreal.EditorLevelLibrary.set_selected_level_actors([actor, ])
+  name = actor.get_actor_label()
+  filename = osp.join(export_dir, f'{name}.gltf')
+  task = unreal.AssetExportTask()
+  task.set_editor_property('selected', True)
+  task.set_editor_property('automated', True)
+  task.set_editor_property('prompt', False)
+  task.set_editor_property('object', world)
+  task.set_editor_property('filename', filename)
+  # unreal.GLTFExporter.set_editor_property('export_task', task)
+  options = unreal.GLTFExportOptions()
+  options.set_editor_property('export_cameras', False)
+  options.set_editor_property('export_lights', False)
+  options.set_editor_property('export_uniform_scale', TRANS_UE_TO_MUJOCO_SCALE)
+  options.set_editor_property('export_unlit_materials', False)
+  options.set_editor_property('export_vertex_colors', False)
+  selected_actors = unreal.Set(unreal.Actor)
+  selected_actors.add(actor)
+  print(f'Writing actor {name} to {filename}...')
+  count = 0
+  N = 10
+  while not unreal.GLTFExporter.export_to_gltf(None, filename, options, selected_actors):
+  # while not unreal.Exporter.run_asset_export_task(task):
+    print(f'Retry {count+1} / {N}')
+    time.sleep(0.5)
+    if count == 10:
+      break
+  else:
+    return True
+  return False
+
+
+PhysicsConstraint = namedtuple(
+  'PhysicsConstraint',
+  ['parent', 'child', 'name', 'range', 'ref', 'axis', 'type']
+)
+MeshComponent = namedtuple('MeshComponent', ['decompose_method', 'decompose_group'])
+
+
+if __name__ == '__main__':
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--scene_path', required=True)
+  parser.add_argument('--actors', default=None,
+                      help='Comma separate list of actor names - all others will be excluded if this option is used')
+  args = parser.parse_args()
+  filename = osp.join(osp.dirname(osp.abspath(__file__)), 'params.yaml')
+  with open(filename, 'r') as f:
+    params = yaml.safe_load(f)
+  # TODO(samarth): access params stuff by attributes
+  export_dir = osp.expanduser(osp.join(args.scene_path, params['common']['GLTF_SCENE_DIR_NAME']))
+  os.makedirs(export_dir, exist_ok=True)
+  include_actors = args.actors.split(',') if args.actors else None
+  
+  filename = osp.join(export_dir, 'actors_information.json')
+  if osp.isfile(filename):
+    with open(filename, 'r') as f:
+      body_properties = json.load(f)
+  else:
+    body_properties = {}
+  
+  # Iterate through the actors
+  for actor in unreal.GameplayStatics.get_all_actors_of_class(world, unreal.Actor):
+    if isinstance(actor, unreal.CameraActor) or isinstance(actor, unreal.SceneCapture2D) or \
```

Comment:
Instead of filtering by type and name, let's filter by path in the World Outliner and assert that we only encounter StaticMeshActors (not inherited) and Actors (not inherited).